internal/utils/flags.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+)
+
+// AppendToFlag is a quick function that appends content to a flag if is already defined
+func AppendToFlag(flags []string, flag, content string) []string {
+	// no content to append to flag
+	if content == "" {
+		return flags
+	}
+
+	var match bool
+	for index, flagEntry := range flags {
+		if strings.Contains(flagEntry, flag) {
```

Comment:
"strings.Contains" might cause issues with the `-mod` or `-p` flags, if one would have also set `-modfile` or `-pkgdir`, so it might better to compare "flagEntry" and the "flag" directly in this case

Reply:
Refactored the function and added a test to verify behaviour. I didn't take into account the cases you mentioned @therecipe